deployment/docker/Makefile

```diff
@@ -169,15 +170,14 @@ package-via-docker-386:
 remove-docker-images:
 	docker image ls --format '{{.Repository}}\t{{.ID}}' | awk '{print $$2}' | xargs docker image rm -f
 
-
 docker-single-up:
-	docker-compose -f deployment/docker/docker-compose.yml up
+	$(DOCKER_COMPOSE) -f deployment/docker/docker-compose.yml up -d
```

Comment:
why adding `-d`?

Reply:
Using `docker compose up` will need to keep opened a terminal window when `docker compose up -d` flag will start and keep working docker containers [in background.](https://docs.docker.com/engine/reference/commandline/compose_up/#options)

Reply:
yes, but that was the original intention. 
Is your concern only about having an extra open tab?

Reply:
Not only that but if you don't use this flag, then running tests on a remote server will already require you to manually run the "docker compose up -d" command instead of using commands from the Makefile.

Reply:
These are scenarios I personally never used. If you think they're important I'd agree. Thanks for explanation.